docs/general-settings.md

```diff
@@ -51,5 +51,10 @@ If you don't specify any value for  **responsiveNavigation**, the buttons remain
 ```
 > **NOTE:** You can translate Luigi internal messages by providing translation for [these keys](../core/src/utilities/defaultLuigiTranslationTable.js).
 * **customSandboxRules** is an array of custom rules for the content in the iframe. You can extend the [Luigi default sandbox rules](https://github.com/SAP/luigi/blob/af1deebb392dcec6490f72576e32eb5853a894bc/core/src/utilities/helpers/iframe-helpers.js#L140) by adding further rules.
+* **iframeCreationInterceptor** is a function called on iframe creation. It gives you full control over the created iframe DOM element. You can modify it to your needs just before it is added to the DOM tree. 
+This function is called with these parameters: 
+  * `iframe` is the iframe DOM element. It is not yet added to the DOM tree, but all attributes are already set.
+  * `viewGroup` is the view group associated with this iframe, if any.
+  * `microFrontendType`, which is `main`, `modal` or `split-view` depending on where it is going to be rendered.
```

Comment:
I would add here a snippet as an example of how you can use it.

Reply:
good idea, done!

Reply:
done!